v22.2/multiregion-overview.md

```diff
@@ -100,6 +100,27 @@ For more information about data domiciling using `PLACEMENT RESTRICTED`, see [Da
 Super regions rely on the underlying [replication zone system](configure-replication-zones.html), which was historically built for performance, not for domiciling. The replication system's top priority is to prevent the loss of data and it may override the zone configurations if necessary to ensure data durability. For more information, see [Configure Replication Zones](https://www.cockroachlabs.com/docs/v21.2/configure-replication-zones#types-of-constraints).
 {{site.data.alerts.end}}
 
+{{site.data.alerts.callout_info}}
+If you are using super regions with the [secondary region](#secondary-regions) feature, the secondary region must be inside or outside of the same super region as the [primary region](set-primary-region.html).
+{{site.data.alerts.end}}
+
+## Secondary regions
+
+<span class="version-tag">New in v22.2:</span> Secondary regions allow you to define a [database region](#database-regions) that will be used for failover in the event your [primary region](set-primary-region.html) goes down.  In other words, the secondary region will act as the primary region if the original primary region fails.
+
+Secondary regions work as follows: when a secondary region is added to a database, a [lease preference](configure-replication-zones.html#lease_preferences) is added to the tables and indexes in that database to ensure that two [voting replicas](configure-replication-zones.html#num_voters) are moved into the secondary region.
+
+This behavior is an improvement over versions of CockroachDB prior to v22.2.  In those versions, when the primary region failed, the [leaseholders](architecture/replication-layer.html#leases) would be transferred to another [database region](#database-regions) at random, which could have negative effects on performance.
+
+For more information about how to use secondary regions, see:
+
+- [`SET SECONDARY REGION`](set-secondary-region.html)
+- [`DROP SECONDARY REGION`](drop-secondary-region.html)
+
+{{site.data.alerts.callout_info}}
+If you are using a secondary region and a [super region](#super-regions), the secondary region must be inside or outside of the same super region as the [primary region](set-primary-region.html).
```

Comment:
Same issue with this callout as the previous comment.

Reply:
Updated using the same new callout (with slight change to link out to super region section)